### DISCUSSION TITLE
Is there an alternative to right-clicking when EarTrumpet pops up?

### DISCUSSION THREAD

**Discussion by Achilles-Heals** [File-New-Project/EarTrumpet#1140] [Q&A] [answered]
**This works already for the EarTrumpet pop-up:**
_arrow up_ = selection moves up
_arrow down_ = selection moves down
_arrow left_ = lowers volume of selected application
_arrow right_ = increases volume of selected application
_tab_ = selection moves down to the next audio output
_shift_+_tab_ = selection moves up to the previous audio output
_m_ = mutes selected application
_._ = also mutes the selected application
_esc_ = escapes EarTrumpet

**What I'm trying to figure out is there a key on the keyboard that mimics what _right-clicking_ does when an application is selected?**

p.s. I'm using a macro pad to use EarTrumpet, but it would be great if I didn't need to use my mouse's _right-click_ to fully utilize EarTrumpet. I've already tried _shift_+_fn_+_f10_ which usually brings up the _right-click_ contextual menu but that doesn't work.

**Comment by riverar** [comment, marked as answer]
Spacebar is the key you're looking for. :smile:

**Reply by Achilles-Heals** [reply]
Thank you so much! I was pressing all the keys trying to figure it out, idk why I didn't think of spacebar. I'm gonna make a macro using that to switch sources with one press.
